pkg/e2e/build.yml

```diff
@@ -0,0 +1,6 @@
+org: linuxkit
+image: kube-e2e-test
+network: true
+disable-content-trust: true
```

Comment:
Lets setup trust for this repo before we merge this PR. I think everything in the linuxkit org really ought to be signed these days.